cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUnitTestUtils.java

```diff
@@ -73,6 +76,8 @@ public static Properties getKafkaCruiseControlProperties() {
     props.setProperty(AnomalyDetectorConfig.TOPIC_ANOMALY_FINDER_CLASSES_CONFIG, NoopTopicAnomalyFinder.class.getName());
     props.setProperty(AnomalyDetectorConfig.SELF_HEALING_GOALS_CONFIG, "");
     props.setProperty(AnalyzerConfig.DEFAULT_GOALS_CONFIG, TestConstants.DEFAULT_GOALS_VALUES);
+    props.setProperty(AnalyzerConfig.BROKER_SET_CONFIG_FILE_CONFIG, brokerSetsDataFile);
+    props.setProperty(AnalyzerConfig.REPLICA_TO_BROKER_SET_MAPPING_POLICY_CLASS_CONFIG, ReplicaToOriginalBrokerSetMappingPolicy.class.getName());
```

Comment:
Just wanted to double check if we are using the non-default config intentionally -- i.e. `TopicNameHashBrokerSetMappingPolicy` -- no change needed if that is intentional

Reply:
Yes this is intentional since default value of ``TopicNameHashReplicaToBrokerSetMappingPolicy`` does group topic based on topic name hashes and not evenness and can violate restrictive tests.